### PR TITLE
Add /travel command and associated files/data

### PR DESCRIPTION
**Pull request by sladecraven**

### CODE REVIEW

File: css/wfrp4e.css [hunk 1]
```diff
@@ -14724,6 +14724,7 @@ button.capture-position {
 .condition-chat,
 .chat-roll,
 .table-click,
+.travel-click,
```
Comment (moo-man) [inline css/wfrp4e.css:14727]: This is specifically for links that show up in chat/editors. Is this intended to work with links? How so?

File: modules/system/dice-wfrp4e.js [hunk 2]
```diff
@@ -820,6 +821,10 @@ export default class DiceWFRP {
       WFRP_Utility.handleTableClick(ev)
     })
 
+    html.on('mousedown', '.travel-click', ev => {
+      TravelDistanceWfrp4e.handleTravelClick(ev)
```
Comment (moo-man) [inline modules/system/dice-wfrp4e.js:825]: This is also for clicking on custom links like tables/pay/conditions/etc. Travel doesn't seem like it fits there

File: css/wfrp4e.css (hunk 1 above)
Comment (sladecraven) [reply css/wfrp4e.css:14727]: yes, when you issue this kind of  command :
/travel -> list of all possible departure towns
/travel town1 -> list of all possibles arrival towns

File: modules/system/dice-wfrp4e.js (hunk 2 above)
Comment (sladecraven) [reply modules/system/dice-wfrp4e.js:825]: Yes, /travel command is issuing links

File: modules/apps/travel-distance-wfrp4e.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        }
+    });
+  }
+
+  /**
+   * Returns a human-readable danger level for the road
+  */
+  static dangerToString( dangerLevel )
+  {
+    console.log("DANGER !!!", dangerLevel);
```
Comment (moo-man) [inline modules/apps/travel-distance-wfrp4e.js:39]: Comment/remove this log too.

Comment (sladecraven) [reply modules/apps/travel-distance-wfrp4e.js:39]: Oops,yes

File: css/wfrp4e.css (hunk 1 above)
Comment (moo-man) [reply css/wfrp4e.css:14727]: Ah, I see the travel-click html now, thanks.

File: modules/apps/travel-distance-wfrp4e.js (hunk 3 above)
Comment (sladecraven) [reply modules/apps/travel-distance-wfrp4e.js]: Fixed

File: modules/apps/travel-distance-wfrp4e.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+      message += `<h3>${game.i18n.localize("TRAVEL.TownOriginPrompt")}</h3>`
+      let uniqTown = {}
+      for ( var travel of this.travel_data) {
+        if ( uniqTown[travel.from] == undefined ) {
+          uniqTown[travel.from] = 1; // Already added in the list
+          message += `<div><a class = "travel-click" data-fromtown="${travel.from}"><i class="fas fa-list"></i> ${travel.from}</a></div>`
+        }
+      }
+    }
+    ChatMessage.create( WFRP_Utility.chatDataSetup(message, "roll") );
```
Comment (moo-man) [inline modules/apps/travel-distance-wfrp4e.js:122]: Can you remove the `modeOverride`  so it takes the default roll mode for visibility?

Comment (sladecraven) [reply modules/apps/travel-distance-wfrp4e.js]: Fixed, with distance tables updated and duration with heavy and light horses

**Comment by sladecraven** [comment]
Distance tables updates, conversion formulas reviewed and duration for light/heavy horses

**Pull request merged**